published/external/xdpapi_helper.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "xdpapi.h"
+
+#define TEST_TIMEOUT_ASYNC_MS 1000
+#define POLL_INTERVAL_MS 10
+
+static
+VOID
+XdpDetach(
+    _In_ VOID *ClientContext
+    )
+{
+    UNREFERENCED_PARAMETER(ClientContext);
+}
+
+NTSTATUS
+InitializeXdpApi(
```

Comment:
We need to think through our verbs, since this is becoming increasingly confusing and unintuitive. We have:

1. Load API
2. Open API
3. Initialize API

Can we eliminate one of these?

Reply:
Eliminate? You mean name the function by connecting the two words from these three?

Reply:
Let's consider renaming:

1. XdpLoadApi -> XdpRegister in kernel mode
2.

Reply:
And then XdpOpenApi can be implemented in xdpapi.h itself.